addons/binding/org.openhab.binding.valloxmv/src/main/java/org/openhab/binding/valloxmv/internal/ValloxMVHandler.java

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.valloxmv.internal;
+
+import java.math.BigDecimal;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import javax.measure.Quantity;
+import javax.measure.Unit;
+import javax.measure.quantity.Dimensionless;
+import javax.measure.quantity.Temperature;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.library.unit.SIUnits;
+import org.eclipse.smarthome.core.library.unit.SmartHomeUnits;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.eclipse.smarthome.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link ValloxMVHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Björn Brings - Initial contribution
+ */
+public class ValloxMVHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(ValloxMVHandler.class);
+    ScheduledFuture<?> updateTasks;
+    ValloxMVWebSocket vows_send;
+
+    /**
+     * Refresh interval in seconds.
+     */
+    private int updateInterval;
+    private long lastUpdate;
+
+    /**
+     * IP of vallox ventilation unit web interface.
+     */
+    public ValloxMVHandler(@NonNull Thing thing) {
+        super(thing);
+    }
+
+    @SuppressWarnings("null")
```

Comment:
What are we suppressing here, is there really no other way or is it a false positive

Reply:
Changed